src/MathConstants.h

```diff
@@ -16,7 +16,10 @@
 #define DEGREES_PER_RAD (57.295779513082)
 
 // macro
-#define NANOMETERS_TO_METERS(x)    (x * 0.000001)
+#define MICROMETERS_TO_METERS(x)    (x * 0.000001)
+#define METERS_TO_MICROMETERS(x)    (x * 1000000)
+#define RAD_TO_DEG_0001(x)          (x * DEGREES_PER_RAD * 10000)
+#define DEG_0001_TO_RAD(x)          ((x / 10000) * RAD_PER_DEGREE)
```

Comment:
These could also be used in `QuatConversion_MpCoordOrient_To_GeomMsgsQuaternion(..)` et al.

But perhaps out of scope for this PR.